module/vcf-input.nf

```diff
@@ -0,0 +1,58 @@
+include { generate_standard_filename } from '../external/pipeline-Nextflow-module/modules/common/generate_standardized_filename/main.nf'
+include { split_VCF_BCFtools } from './common' addParams(
+    keep_input_prefix: true
+    )
+include { rename_samples_BCFtools } from './common' addParams(
+    keep_input_prefix: true
+    )
```

Comment:
Any reason to hard-code `keep_input_prefix` here despite having a parameter for it?

Reply:
As mentioned above, it's not a parameter the user should be changing.  I added it to generalize the processes to work for both types of input.  Is there a better way to do this?